source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,152 @@
+	const {repository} = await api.v4(`
+		repository() {
+			defaultBranchRef {
+				target {
+					...on Commit {
+						history(first:100, path: "source/features/${fileName}") {
+							nodes {
+								messageHeadline
+								messageHeadlineHTML
+								commitUrl
+								committedDate
+							}
+						}
+					}
+				}
+			}
+		}
+	`);
+	return repository.defaultBranchRef.target.history.nodes;
+}, {
+	maxAge: {days: 1},
+	staleWhileRevalidate: {days: 4},
+	cacheKey: ([fileName]): string => __filebasename + ':' + fileName
+});
+
+function getCommitUrl(commit: Commit): string {
+	const [, pullRequestUrl] = /<a[^>]+href="([^"]+)">/.exec(commit.messageHeadlineHTML) ?? [];
+	return pullRequestUrl ?? commit.commitUrl;
+}
+
+async function getHistoryDropdown(featureName: string): Promise<Element> {
+	let history = await getFeatureHistory(featureName + '.tsx');
+
+	if (history.length === 0) {
+		// Feature might be CSS-only
+		history = await getFeatureHistory(featureName + '.css');
+	}
+
+	const filteredHistory = history.filter((commit: Commit) => !/^Meta|^Document|^Readme|^Lint|^Update.+dependencies/.test(commit.messageHeadline));
+	const skippedCommitsCount = history.length - filteredHistory.length;
+
+	const historyUrl = new GitHubURL(location.href);
+	historyUrl.assign({route: 'commits'});
+
+	return (
+		<details className="dropdown details-reset details-overlay d-inline-block">
+			<summary className="text-gray d-inline" aria-haspopup="true">
+				Feature history
+				<div className="dropdown-caret ml-1"/>
+			</summary>
+
+			<div className="dropdown-menu dropdown-menu-s" style={{width: 400}}>
+				<ul className="overflow-y-auto" style={{maxHeight: '60vh'}}>
+					{filteredHistory.map(commit => (
+						<li>
+							<a className="dropdown-item" href={getCommitUrl(commit)} title={commit.messageHeadline}>
+								{parseBackticks(commit.messageHeadline)}
+								<div className="text-small">
+									<relative-time datetime={commit.committedDate}/>
+								</div>
+							</a>
+						</li>
+					))}
+					{skippedCommitsCount > 0 && (
+						<>
+							<li className="dropdown-divider" role="separator"/>
+							<li>
+								<a className="dropdown-item" href={String(historyUrl)}>
+									View full history (+{skippedCommitsCount} commits)
+								</a>
+							</li>
+						</>
+					)}
+				</ul>
+			</div>
+		</details>
+	);
+}
+
+function getConversationsLink(featureName: string): Element {
+	const searchUrl = new URL('https://github.com/sindresorhus/refined-github/issues');
+	searchUrl.searchParams.set('q', `"${featureName}" sort:updated-desc`);
+
+	return <a className="ml-3" href={String(searchUrl)}>Conversations</a>;
+}
+
+async function init(): Promise<void | false> {
+	const [, currentFeature] = /features\/([^.]+)/.exec(location.pathname)!;
+	const {id, description, screenshot} = __featuresMeta__.find(feature => feature.id === currentFeature) ?? {};
+	if (!description) {
+		return false;
+	}
```

Comment:
```suggestion
	const feature = __featuresMeta__.find(feature => feature.id === currentFeature);
	if (!feature) {
		return false;
	}

	const {id, description, screenshot} = feature;
```

This way `id` will never be `string | undefined`